pluto/src/vis/diagram/edge/connector/connector.ts

```diff
@@ -118,6 +119,11 @@ export interface Segment {
   length: number;
 }
 
+export const segmentZ = z.object({
```

Comment:
If you're going to define a zod type for this, make the segment interface this instead:

`type Segment = z.infer<typeof segmentZ>`